Apps/WebClient/src/NewClientApp/src/stores/navbar.ts

```diff
@@ -0,0 +1,48 @@
+﻿import { computed, ref } from "vue";
+import { useAppStore } from "@/stores/app";
+import { ILogger } from "@/services/interfaces";
+import { container } from "@/ioc/container";
+import { SERVICE_IDENTIFIER } from "@/ioc/identifier";
+import { defineStore } from "pinia";
+
+export const useNavbarStore = defineStore("navbar", () => {
+    const logger = container.get<ILogger>(SERVICE_IDENTIFIER.Logger);
+
+    const appStore = useAppStore();
+
+    const isSidebarOpenField = ref<boolean>();
+    const isSidebarAnimating = ref<boolean>(false);
+    const isHeaderShown = ref<boolean>(true);
+
+    const isSidebarOpen = computed(() => {
+        if (isSidebarOpenField.value === null) {
+            return !appStore.isMobile;
+        }
+        return isSidebarOpenField.value;
+    });
```

Comment:
Since the type of the field is now `boolean | undefined` rather than `boolean | null`, the comparison should be against `undefined` rather than `null`.  It also means this can be simplified.

```suggestion
    const isSidebarOpen = computed(() => isSidebarOpenField.value ?? !appStore.isMobile);
```

Reply:
Good suggestion.